.github/workflows/main.yml

```diff
@@ -4,12 +4,13 @@ on:
   push:
     branches: [main]
   pull_request:
-    types: [opened, synchronize]
+    types: [opened, synchronize, reopened]
 
 concurrency: ${{ github.workflow }}-${{ github.ref }}
 
 jobs:
   build:
+    if: github.event.pull_request.head.ref != 'changeset-release/main' # This should skip the build step for the Pull requests that are opened from the branch changeset-release/main.
```

Comment:
This `build` step will still happen once the PR gets merged into `main` though, right?

Does this also mean that this `build` job has been entirely unnecessary when our automation creates version bump PRs?

Reply:
yes, on merge `github.event.pull_request.head.ref` should be `undefined` thus this expression will be `true`.

And yes, since version PR only updates versions we don't need to run this job, as purpuse of the Build job is to only check the code.

Reply:
Okay. I think this makes sense to me now. Thanks for helping me understand.